go.mod

```diff
@@ -20,11 +20,12 @@ require (
 	github.com/rjeczalik/notify v0.9.3-0.20210809113154-3472d85e95cd
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.7.0
+	github.com/valyala/fasthttp v1.37.0
```

Comment:
Привет! Мы решили отказаться от стандартного клиента?

Reply:
Because many small requests to the ES server are expected, fasthttp is well optimized for this, you can see the benchmark results. Are there any reasons why we should use net/http?